easybuild/scripts/rpath_wrapper_template.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
because iterating of the command line arguments in bash is damn nasty to implement imho, it can be done way cleaner in a Python script (which is easier to unit test too).